testsuite/integration-arquillian/tests/base/testsuites/database-suite

```diff
@@ -0,0 +1,26 @@
+AccountRestServiceTest
+AuthorizationCodeTest
+AuthorizationTest
+ClientRegistrationTest
+EventStoreProviderTest
+ExportImportTest
+GeneratedRsaKeyProviderTest
+KcOidcBrokerTest
+LDAPUserLoginTest
+LoginTest
+PasswordPolicyTest
+RequiredActionUpdateProfileTest
+SSOTest
+SamlClientTest
+TransactionsTest
+UserProfileTest
+org.keycloak.testsuite.admin.*
+org.keycloak.testsuite.admin.authentication.*
+org.keycloak.testsuite.admin.client.*
+org.keycloak.testsuite.admin.concurrency.*
+org.keycloak.testsuite.admin.event.*
+org.keycloak.testsuite.admin.group.*
+org.keycloak.testsuite.admin.partialexport.*
+org.keycloak.testsuite.admin.partialimport.*
+org.keycloak.testsuite.admin.realm.*
+org.keycloak.testsuite.admin.userprofile.*
```

Comment:
Should `org.keycloak.testsuite.admin` contain all CRUD operations for admin REST API, I'd very much prefer `org.keycloak.testsuite.admin.**` rather than this list, just like suggested previously by @mposolda

Reply:
`**` isn't an opinion IMO as there's too many tests in the authz side of things. As a follow-up we can re-organize the authz stuff, and add some more testing in there, but we don't need this absolutely perfect for the first iteration. It can and should be tuned after the PR is merged.

Reply:
I believe `**` is aligned with the [comment above](https://github.com/keycloak/keycloak/pull/15862#discussion_r1043347952) which states that this refactoring should be done prior to merging this PR. I am thus confused by your agreement to that comment and at the same time stating that it should be done as a follow up here.

I believe it should be done prior to this PR, and then my comment would be still valid

Reply:
Changed to **

Reply:
Thank you, resolving this.